src/lib.rs

```diff
@@ -87,10 +87,9 @@ pub fn run_with(cmd_args: Option<&[&str]>) -> Result<(), String> {
 
     // We need a minimum of INFO level for vaults verbosity,
     // since the genesis vault logs the contact info at INFO level
-    let verbosity = format!("-{}", "v".repeat(2 + args.vaults_verbosity as usize));
```

Comment:
We should still need this

Reply:
Hmm, fat fingers as i worked or github's UI is doing some fun things. Will fix now